standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/client/TestGetPartitions.java

```diff
@@ -416,7 +417,7 @@ public void testGetPartitionsByNamesNullDbName() throws Exception {
       createTable3PartCols1Part(client);
       client.getPartitionsByNames(null, TABLE_NAME, Lists.newArrayList("yyyy=2000/mm=01/dd=02"));
       fail("Should have thrown exception");
-    } catch (NullPointerException | TTransportException e) {
+    } catch (NullPointerException | TTransportException | MetaException e) {
```

Comment:
This is again an HMS API change which should be avoided

Reply:
coalesce with above as 1 single test issue.